frameworks/agno/from_agno/agents/TravelPlannerAgent/query_parser.py

```
@@ -0,0 +1,87 @@
+            messages=[
+                {"role": "system", "content": system_prompt},
+                {"role": "user", "content": user_query}
+            ],
+            temperature=0.1,
+            max_tokens=300
+        )
+
+        content = response.choices[0].message.content.strip()
+        parsed = json.loads(content)
+        logger.info("Successfully parsed travel query: %s", parsed)
+        return parsed
```

Ruff (0.12.2)

Consider moving this statement to an `else` block

(TRY300)